src/agent/input-tester/src/crash_detector.rs

```diff
@@ -160,48 +152,10 @@ impl<'a> CrashDetectorEventHandler<'a> {
             stderr_buffer: vec![],
             debugger_output: String::new(),
             exceptions: vec![],
-            id_to_block: None,
```

Comment:
Splitting up the new `Recorder` from its debugger event handler, and will splice it into this struct.

Reply:
Beyond what we discussed, this will also be useful for other fuzzers like radamsa.